src/bundle/Resources/public/scss/functions/_calculate.rem.scss

```diff
@@ -3,5 +3,5 @@
 @function calculateRem($size) {
     $remSize: div($size, $base-font-size);
 
-    @return #{$remSize}rem;
+    @return $remSize + 0rem;
```

Comment:
The previous version was actually not converting values to `rem` measurement in SCSS. Change required for `$modal-inner-padding`;